src/app/Classes/Attributes/Meta.php

```diff
@@ -11,6 +11,7 @@ class Meta
         'disabled', 'readonly', 'hidden', 'source', 'format', 'time', 'rows',
         'placeholder', 'trackBy', 'label', 'tooltip', 'symbol', 'precision',
         'thousand', 'decimal', 'positive', 'negative', 'zero', 'resize',
+        'translateFields',
```

Comment:
let's rename `translateFields` to `translated`.

Add a note to the docs as well. Make sure you refer `options`, not `fields`